cs-couchbase/src/main/java/io/cloudslang/content/couchbase/utils/InputsUtil.java

```diff
@@ -65,13 +69,13 @@ public static HttpClientInputs getHttpClientInputs(String username, String passw
         HttpClientInputs httpClientInputs = new HttpClientInputs();
         httpClientInputs.setMethod(method);
         httpClientInputs.setAuthType(BASIC);
-        httpClientInputs.setQueryParamsAreURLEncoded(FALSE);
+        httpClientInputs.setQueryParamsAreURLEncoded(valueOf(FALSE));
         httpClientInputs.setUsername(username);
         httpClientInputs.setPassword(password);
 
-        httpClientInputs.setTrustAllRoots(valueOf(getEnforcedBooleanCondition(trustAllRoots, true)));
-        httpClientInputs.setKeepAlive(valueOf(getEnforcedBooleanCondition(keepAlive, true)));
-        httpClientInputs.setUseCookies(valueOf(getEnforcedBooleanCondition(useCookies, true)));
+        httpClientInputs.setTrustAllRoots(valueOf(getEnforcedBooleanCondition(trustAllRoots, FALSE)));
+        httpClientInputs.setKeepAlive(valueOf(getEnforcedBooleanCondition(keepAlive, TRUE)));
```

Comment:
do we need these tricks with objects? maybe we can just rely on boxing

Reply:
You're right... I'll find a way to simplify it.